core-bundle/src/Image/Studio/FigureBuilder.php

```diff
@@ -527,7 +527,7 @@ private function doBuild(): Figure
         $settings = clone $this;
 
         $imageResult = $this->locator
-            ->get(Studio::class)
+            ->get('contao.image.studio')
```

Comment:
`Studio::class` should get an autowiring alias and this change can then be reverted.